packages/mui-base/src/SelectUnstyled/SelectUnstyled.test.tsx

```diff
@@ -103,6 +103,26 @@ describe('SelectUnstyled', () => {
       });
     });
 
+    ['Enter', ' '].forEach((key) => {
+      it(`does not close the multiselect dropdown when the "${key}" key is pressed`, () => {
```

Comment:
```suggestion
      it(`does not close the multiselect dropdown when the "${key}" key is pressed if the multiple prop is applied`, () => {
```

Reply:
Isn't the "multiselect dropdown" part enough?

Reply:
I missed the multiselect part :)